composer.json

```diff
@@ -132,6 +132,7 @@
         "symfony/translation": "^5.4.21 || ^6.0",
         "symfony/translation-contracts": "^2.5",
         "symfony/twig-bundle": "^5.4.21 || ^6.0",
+        "symfony/ux-twig-component": "^2.9.1",
```

Comment:
Should we upgrade the version to `^2.11.2` in accordance with the symfony.lock file?

Reply:
I checked this and it won't change too much. On newer version of PHP it'll install `2.11.2`, on older ones `2.9.1` to is doesn't change anything what we keep in the `symfony.lock` as it is used only by recipes.

Reply:
Thanks for watching.